src/dmd/astbase.d

```diff
@@ -4470,6 +4470,7 @@ struct ASTBase
         size_t len;         // number of code units
         ubyte sz = 1;       // 1: char, 2: wchar, 4: dchar
         char postfix = 0;   // 'c', 'w', 'd'
+        char interpolate = 0;
```

Comment:
Is there a reason this isn’t a `bool`?

Reply:
probably should be `ubyte`, but it's to match the `C++` ABI

Reply:
nevermind, I see `bool` is used elsewhere in D/C++ files...switching to `bool`